src/tests/Display.test.js

```diff
@@ -6,9 +6,21 @@ import Display from '../components/Display/Display';
 
 Enzyme.configure({ adapter: new Adapter() });
 
+// describe('<Display />', () => {
+//   it('renders without crashing', () => {
+//     const div = document.createElement('div');
+//     ReactDOM.render(<Display />, div);
+//   });
+// });
 describe('<Display />', () => {
   it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<Display />, div);
+      shallow(<Button />);
   });
-});
+
```

Comment:
Consider testing the props passed to the component, and the Text it renders when you pass the correct prop to the Display.